rustls/Cargo.toml

```diff
@@ -17,6 +17,7 @@ rustversion = { version = "1.0.6", optional = true }
 
 [dependencies]
 aws-lc-rs = { version = "1.5", optional = true }
+hashbrown = { version = "0.13", optional = true }
```

Comment:
Did you consider using 0.14 here? Is there a reason to avoid that?

Reply:
yes, 0.14 requires a higher msrv than what rustls currently has.

Reply:
Ahh yup, that would be a good reason :-) Thanks